storybook/index.js

```diff
@@ -12,6 +13,20 @@ const StorybookUIRoot = getStorybookUI({
   asyncStorage: require('@react-native-community/async-storage').default || require('react-native').AsyncStorage || null
 });
 
-AppRegistry.registerComponent('%APP_NAME%', () => StorybookUIRoot);
+const styles = StyleSheet.create({
+  AndroidSafeArea: {
+    flex: 1,
+    backgroundColor: 'white',
+    paddingTop: Platform.OS === 'android' ? StatusBar.currentHeight : 0
+  }
+});
+
+const SafeStorybook = () => (
+  <SafeAreaView style={styles.AndroidSafeArea}>
```

Comment:
Might be worth commenting on why this is in place? Is there no iOS safe area? Things worth covering in comments at least perhaps?

Reply:
👍  `SafeAreaView` handles iOS inherently - that is what it's for. I'll add a comment

Reply:
Awesome - Thanks. I didn't know about that. The notch was quite new when I was last doing full-time React Native 👍